crystal/athena/src/server.cr

```diff
@@ -1,5 +1,7 @@
 require "athena"
 
+Log.setup :none
```

Comment:
@Blacksmoke16 I prefer to avoid an`info` **strategy** `here`, as for `lucky`

Reply:
I agree that we are **NOT** benchmarking logging feature. However
+ to have realistic results, we **SHOULD NOT** tune implementation for performances. I mean to ensure results accuracy, we'd rather have **production-grade** implementation, and I never seen any application deployed in **production** without logging
+ The most important thing is to compare `crystal` **frameworks** with the same level of logging

e`crystal

Reply:
> * The most important thing is to compare `crystal` **frameworks** with the same level of logging

Yes, which should be `Log::Severity::None`.  Having logging enabled during benchmarks skews the results.  Framework A logs 1 thing per request, Framework B logs 2 things per request; Framework B is now most likely slower than Framework A just because it logs more.  Since you can't standardize how much each framework logs, the best course of action is to just standardize that it should be disabled.

Reply:
I think it should be standardize as all have **info** logging level, since it is that the recommendation for any **production** app.

Btw, no opposed to any of.

Please vote in https://github.com/the-benchmarker/web-frameworks/issues/2622, this issue is blocking that's why we should externalize this feature (I'll take your contribution as-is @Blacksmoke16)